subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/dependencysubstitution/DefaultDependencySubstitutions.java

```diff
@@ -256,20 +256,6 @@ public Substitution using(ComponentSelector notation) {
                 }
                 return this;
             }
-
-            @Override
-            @Deprecated
-            public void with(ComponentSelector substitute) {
-                // Do not nag for 7.1 as this introduces a performance regression with Android plugin
-                // Revisit when upgrading Android plugin
-                /*DeprecationLogger.deprecateMethod(Substitution.class, "with(ComponentSelector)")
-                    .replaceWith("using(ComponentSelector)")
-                    .willBeRemovedInGradle8()
-                    .withUpgradeGuideSection(7, "dependency_substitutions_with")
-                    .nagUser();*/
-
```

Comment:
Since we don't have the nagging, was this ever removed from Android?

Reply:
I believe so, no smoke tests are failing.